docs/distribution/README.md

```diff
@@ -0,0 +1,222 @@
+# OCI Distribution
+
+We introduce new REST APIs in the registry to support storing signature objects together with the target artifacts, and retrieving them for verification.
+
+## Table of contents
+1. [Sample APIs for retrieving signatures](#Sample-APIs-for-retrieving-signatures)
+2. [Sample APIs for storing signatures](#Sample-APIs-for-storing-signatures)
+3. [Implementation](#Implementation)
+
+## Sample APIs for retrieving signatures
+
+The list of signatures for an artifact can be retrieved from the registry.
+
+### _Get a list of paginated signatures by tag_
+
+### Request
+`GET http://localhost:5000/v2/hello-world/manifests/v1.0/signatures/`
+
+### Response
+
+```json
+{
+    "tag": "v1.0",
+    "digest": "sha256:80559bf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cbc042",
+    "@nextLink": "{opaqueUrl}",
+    "signatures": [
+        "sha256:2235d2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49cc77c",
+        "sha256:3335d2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49cc88d"
+    ]
+}
+```
+
+### _Get a list of paginated signatures by digest_
+
+### Request
+`GET http://localhost:5000/v2/hello-world/manifests/sha256:80559bf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cbc042/signatures/`
+
+### Response
+
+```json
+{
+    "digest": "sha256:80559bf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cbc042",
+    "@nextLink": "{opaqueUrl}",

```

Comment:
It's not clear what the value of nextLink would be, and how someone starts with page 3.
Can you elaborate a bit more in the example?

Reply:
From previous discussions, I got the impression that folks are less keen on client driven paging. With opaque nextLinks generated by the server, starting on page 3 is not really an option